### ISSUE TITLE
Improve CRD nav menu UI

### ISSUE THREAD

**Issue by jim-docker** [kontena/lens#7179]
**What would you like to be added**:
**Why is this needed**:
Comment from customer through support channel:
>Good morning:  UI Suggestion.  Custom resources have become very popular. The long list of custom resources is not very user friendly.  I find my scrolling through a long list of CRs to find the right entry. There needs to be more visual distinction between the list item and its sub-items.  Also, sorting by name ends up separating related CRs in to different parts of the list - making it difficult to use. Maybe a separation between groups?  Not sure what to do, only that it's not. working the way its designed now.  No feedback from you is needed.  Just wanted to add this to suggestions for improvement or consideration.

related: https://github.com/lensapp/lens/issues/3593, #883

**Environment you are Lens application on:**

- Kubernetes distribution: any/all
- Desktop OS: any/all

**Comment by OliverRC** [comment]
I agree I just moved from the Deployment resource to a CRD from Argo Rollouts but now the Rollouts and CRD are way down at the bottom.

It would be amazing if your could "Favourite" certain CRD's and even give them Friendly headings/
